Cargo.toml

```diff
@@ -18,3 +19,8 @@ exclude = [
 [patch.crates-io]
 tracing = {git = "https://github.com/tokio-rs/tracing/", branch = "v0.1.x"}
 tracing-core = {git = "https://github.com/tokio-rs/tracing/", branch = "v0.1.x"}
+
+[profile.release]
+lto = true
+opt-level = 3
+panic = "unwind"
```

Comment:
I've enabled some optimizations for release build and added `panic = "unwind"`, which we rely on in #419 for temporarily catching panics from PoS native VP, which is ran in a new thread